app/models/manageiq/providers/amazon/inventory/targets.rb

```diff
@@ -1,4 +1,9 @@
 class ManageIQ::Providers::Amazon::Inventory::Targets < ManageIQ::Providers::Amazon::Inventory
```

Comment:
Same here, why is that a plural class name? And why does it inherit from `Inventory` ?